edge/services/helloworld/README.md

```diff
@@ -176,180 +205,53 @@ hzn exchange service listpolicy <published-helloworld-service-name>
+  ```bash
+  sudo tail -f /var/log/syslog | grep helloworld[[]
+  ```
 
-### Preconditions for Developing Your Own Service
+  on **Mac**:
 
-1. First, go through the steps in the section above to run the IBM helloworld service on an edge node.
-2. Get a docker hub id at https://hub.docker.com/ , if you don't already have one. (This example is set up to store the docker image in docker hub, but by modifying DOCKER_IMAGE_BASE you can store it in another registry.) Login to the docker registry using your id:
-```
-echo 'DOCKER-PASSWORD' | docker login -u DOCKER-HUB-ID --password-stdin
-```
+  ```bash
+  sudo docker logs -f $(sudo docker ps -q --filter name=helloworld)
+  ```
 
-3. If you have the HZN_ORG_ID environment variable set from previous work, unset it (in a moment this value will now come from `horizon/hzn.json`):
-```
-unset HZN_ORG_ID
-```
-4. Cd to the directory in which you want to create your new service and then:
-```
-hzn dev service new -o <org-id> -s <service-name> -i <docker-image-base>
-# E.g.:  hzn dev service new -o bp@someemail.com -s bp.helloworld -i brucemp/bp.helloworld
-```
-
-5. As part of the above section "Using the Hello World Example Edge Service", you created your Exchange user credentials and edge node credentials. Ensure they are set and verify them:
-```
-export HZN_EXCHANGE_USER_AUTH="iamapikey:PUT-YOUR-API-KEY-HERE"
-hzn exchange user list
-export HZN_EXCHANGE_NODE_AUTH="PUT-ANY-NODE-ID-HERE:PUT-ANY-NODE-TOKEN-HERE"
-hzn exchange node confirm
-```
-
-### Building and Publishing Your Own Version of the Hello World Example Edge Service
-
-1. Edit `service.sh`, for example changing "Hello" to "Hey there"
-    - Note: this service is a shell script simply for brevity, but you can write your service in any language.
-2. Build the hello world docker image:
-```
-make
-```
-3. Test the service by having Horizon start it locally:
-```
-hzn dev service start -S
-```
-
-4. Check that the container is running:
-```
-sudo docker ps 
-```
-
-5. See the docker container running and look at the output:
-
-	on **Linux**:
-	```
-	tail -f /var/log/syslog | grep helloworld[[]
-	```
-
-	on **Mac**:
-	```
-	docker logs -f $(docker ps -q --filter name=helloworld)
-	``` 
-
-6. See the environment variables Horizon passes into your service container:
-```
-docker inspect $(docker ps -q --filter name=helloworld) | jq '.[0].Config.Env'
-```
-7. Stop the service:
-```
-hzn dev service stop
-```
-8. Create a service signing key pair in `~/.hzn/keys/` (if you haven't already done so):
-```
-hzn key create <my-company> <my-email>
-```
-9. Have Horizon push your docker image to your registry and publish your service in the Horizon Exchange and see it there:
-```
-hzn exchange service publish -f horizon/service.definition.json
-hzn exchange service list
-```
-10. Publish your edge node deployment pattern in the Horizon Exchange and see it there:
```

Comment:
I suggested a diff sentence format for this earlier.

Reply:
notice those are lines shaded red, which means they are being deleted in this PR